src/humancellatlas/data/metadata/api.py

```diff
@@ -582,8 +583,11 @@ class ImagingPreparationProtocol(Protocol):
     pass
 
 
-@dataclass
+@dataclass(init=False)
 class ManifestEntry:
+    # FIXME: would be MutableJSON is Azul types were available
+    # https://github.com/DataBiosphere/hca-metadata-api/issues/13
```

Comment:
```suggestion
    # FIXME: would be MutableJSON is Azul types were available
    #         https://github.com/DataBiosphere/hca-metadata-api/issues/13
```

to indicate that second line is continuation of FIXME. PyCharm supports that and my previous review included it. Please pay more attention to the details.